internal/acceptance_test/resource_instances_test.go

```diff
@@ -41,7 +37,7 @@ func TestAccResourceInstanceCreate(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
-		CheckDestroy: resource.ComposeTestCheckFunc(testVmaasInstanceDestroy("hpegl_vmaas_instance.tf_instance")),
```

Comment:
checkResourceDestroy seems specific to router. does this work for instance as well ?

Reply:
Yes, correct. I updated checkResourceDestroy function